sdk/src/main/java/io/opentelemetry/sdk/trace/samplers/ProbabilitySampler.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.trace.samplers;
+
+import com.google.auto.value.AutoValue;
+import com.google.common.base.Preconditions;
+import io.opentelemetry.trace.Sampler;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.SpanContext;
+import io.opentelemetry.trace.SpanId;
+import io.opentelemetry.trace.TraceId;
+import java.util.List;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.Immutable;
+
+/**
```

Comment:
`@since 0.1.0` is missing here.

Reply:
Seems none of the SDK classes have the `@since` tag. I'll wait to see our decision on https://github.com/open-telemetry/opentelemetry-java/issues/344.